doc/source/_toc.yml

```
@@ -283,7 +283,7 @@ parts:
                 - file: cluster/cluster_under_construction/ray-clusters-on-kubernetes/examples/ml-example.md
             - file: cluster/cluster_under_construction/ray-clusters-on-kubernetes/references
           - file: cluster/cluster_under_construction/ray-clusters-on-vms/index
-            title: Ray Clusters on VMs
+            title: Deploying a Ray Cluster on VMs
```

```suggestion
            title: Deploy a Ray Cluster on VMs
```
Keeping the naming consistent with the other page.

---

Nice catch.. I'm probably going too fast if I missed this